src/test/java/edu/harvard/dbmi/avillach/dictionary/facet/FacetQueryGeneratorTest.java

```diff
@@ -0,0 +1,44 @@
+package edu.harvard.dbmi.avillach.dictionary.facet;
+
+import edu.harvard.dbmi.avillach.dictionary.filter.Filter;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+class FacetQueryGeneratorTest {
+
+    @Autowired
+    FacetQueryGenerator generator;
+
+    @Test
+    void shouldaaa() {
+        Filter jim = new Filter(List.of(
+            new Facet("phs000007", "", "", null, null, "study_ids_dataset_ids", null),
+            new Facet("PhenX", "", "", null, null, "nsrr_harmonized", null)
+        ), "");
+        String query = generator.createFacetSQLAndPopulateParams(jim, new MapSqlParameterSource());
+        System.out.println(query);
```

Comment:
Is this asserting nothing? Does it mean this test will always pass? (Not familiar enough with the testing pattern to tell immediately.)